client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/item-row.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import TextField from 'woocommerce/woocommerce-services/components/text-field';
+import {
+	setCustomsItemDescription,
+	setCustomsItemTariffNumber,
+	setCustomsItemOriginCountry,
+} from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+import CountryDropdown from 'woocommerce/woocommerce-services/components/country-dropdown';
+
+const TariffCodeTitle = localize( ( { translate } ) =>
+	<span>{ translate( 'Tariff Code' ) } (<a href="https://hts.usitc.gov/"
+		target="_blank" rel="noopener noreferrer">{ translate( 'look up' ) }</a>)</span>
+);
+
+const ItemRow = ( props ) => {
+	const {
+		errors,
+		packageId,
+		productId,
+		translate,
+		description,
+		defaultDescription,
+		tariffNumber,
+		originCountry,
+		countriesData,
+	} = props;
+
+	return <div className="customs-step__item-row">
+		<TextField
+			id={ packageId + '_' + productId + '_description' }
+			className="customs-step__item-description-column"
+			title={ translate( 'Description' ) }
+			value={ description }
+			placeholder={ defaultDescription }
+			updateValue={ props.setCustomsItemDescription }
+			error={ errors.description } />
+		<CountryDropdown
+			id={ packageId + '_' + productId + '_originCountry' }
+			className="customs-step__item-country-column"
+			title={ translate( 'Origin country' ) }
+			value={ originCountry }
+			updateValue={ props.setCustomsItemOriginCountry }
+			countriesData={ countriesData } />
+		<TextField
```

Comment:
Just a thought: might want to put tariff code next to the description, since they are both intrinsic to the item / seemingly more closely related to each other than either is to origin.

Also, a future enhancement could be to have a little search icon on each tariff code field that links to the lookup query based on the description ([for instance](https://hts.usitc.gov/?query=snow%20crab)).

Reply:
> Just a thought: might want to put tariff code next to the description, since they are both intrinsic to the item / seemingly more closely related to each other than either is to origin.

Done in https://github.com/Automattic/wp-calypso/pull/24974/commits/86823478fd077edf7ee9fb3afa24a3507b46c0cc

> Also, a future enhancement could be to have a little search icon on each tariff code field that links to the lookup query based on the description (for instance).

Yeah, there's definitely some things we can do to help the merchant in filling the tariff code, but I want to get this shipped first :/